module/chunks/chunkVerifier_test.go

```diff
@@ -271,6 +271,14 @@ func GetBaselineVerifiableChunk(t *testing.T, script string, system bool) *verif
 
 	f, _ := completeLedger.NewLedger(&fixtures.NoopWAL{}, 1000, metricsCollector, zerolog.Nop(), completeLedger.DefaultPathFinderVersion)
 
+	compactor := fixtures.NewNoopCompactor(f)
+	<-compactor.Ready()
+
+	defer func() {
+		<-f.Done()
+		<-compactor.Done()
+	}()
+
```

Comment:
do we need this for this test? I don't think we use the compactor in this case.

Reply:
> do we need this for this test?

Yes.  We need a compactor running when `Ledger.Set` is called, even if we're not testing checkpointing or WAL segments.  `Ledger.Set` communicates with Compactor over channels.  If compactor isn't running, `Ledger.Set` would block.  

In old implementation, an anonymous goroutine is launched for each update, which calls `wal.RecordUpdate()`.  With `NoopWAL` used in tests, no WAL update is written.

In this PR, the anonymous goroutine is replaced by communicating with `Compactor` over channels.  `NoopCompactor` is used in tests for minimum communication.